CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+project(CMaizeSuperbuild)
+
+include(CTest)
+include(ExternalProject)
+include(FetchContent)
```

Comment:
You don't need `FetchContent`. `ExternalProject_Add` can get souce code. As a disclaimer I think way down the road we'll actually want to use `FetchContent` to get the source code (it'll make it easier to build offline, and for the user to override the default dependency version), but let's not worry about that until we have the superbuild in place.